blackbox/docs/sql/reference/alter_table.txt

```diff
@@ -128,12 +128,23 @@ Can be used to rename a table, while maintaining its schema and data. During
 this operation the table will be closed, and all operations upon the table will
 fail until the rename operation is completed.
 
+.. _alter_table_reroute:
+
 ``REROUTE``
 -----------
 
 The ``REROUTE`` command provides various options to manually control the
 allocation of shards. It allows to enforce explicit allocations, cancel them and
```

Comment:
"it allows the enforcement of explicit allocations, cancellations and the moving of shards between nodes in a cluster."